crates/router/src/core/payment_methods/surcharge_decision_configs.rs

```diff
@@ -403,21 +403,8 @@ pub async fn perform_surcharge_decision_management_for_saved_cards(
         backend_input.payment_method.payment_method_type =
             customer_payment_method.payment_method_type;
 
-        let card_network = match &customer_payment_method.payment_method_data {
-            Some(api_models::payment_methods::PaymentMethodListData::Card(card)) => card
-                .scheme
-                .as_ref()
-                .map(|scheme| {
-                    scheme
-                        .clone()
-                        .parse_enum("CardNetwork")
-                        .change_context(ConfigError::DslExecutionError)
-                })
-                .transpose()?,
-            _ => None,
-        };
-
-        backend_input.payment_method.card_network = card_network;
+        // check in review
+        backend_input.payment_method.card_network = None;
```

Comment:
The code is removed here, is this a expected change?

Reply:
card_network is removed from the request in v2, so need to decide what to do as it won't be present in here